src/main.cpp

```diff
@@ -445,7 +445,7 @@ int app( int argc, char** argv ) {
 
     // Check if output is a tty before dumping binary data to it
     if ( isatty( fileno( stdout ) ) && !maimOptions->savepathGiven ) {
-        std::cout << HELP_MESSAGE << std::endl;
+        std::cout << "Please provide an output path or redirect stdout to a file" << std::endl;
```

Comment:
I don't think this is the best change. Maybe it is better to print a suggestion to run `maim --help`, too? Would make things more obvious for end users.

Reply:
I just know that the current solution did not help me at all, because I expected it to work from previous versions and all it did was spit out the help repeatedly, even though it lists the filename as optional.

I'm open to suggestions, but I think that this PRs output is much more user friendly than just printing the help without actually telling the user what was wrong. Usually the user doesn't run commands he thinks aren't valid, if the user thinks it should work, just printing the help over and over doesn't help, the user needs to know what he did wrong that main doesn't understand.

Reply:
Oh, you did not understand what do I mean.I believe the best solution is to print the error message you have suggested **and** tell user he can run `maim --help`, too.
Eventually it may look like this:
```
$ maim
Please provide an output path or redirect stdout to a file
Run maim --help for more information
$
```